site/src/docs/foundation/guidelines/server-side-rendering.mdx

```diff
@@ -0,0 +1,171 @@
+---
+slug: "/foundation/guidelines/server-side-rendering"
+title: "Server-side rendering"
+navTitle: 'Server-side rendering'
+---
+import { Accordion, Notification } from 'hds-react';
+import LeadParagraph from '../../../components/LeadParagraph';
+
+# Server-side rendering
+
+<LeadParagraph>
+  The server-side rendering support of the Helsinki Design System makes the user's landing experience to pages smooth without
+  flashes of unstyled content.
+</LeadParagraph>
+
+## What is server-side rendering?
+
+Server-side rendering (SSR) is the process where the rendering of HTML pages is done on the server-side. The fully rendered HTML
+document is then sent to the browser. The rendering of the HTML can happen at build time (Static site generation
+or pre-rendering), or during an HTTP request. The alternative to SSR is client-side rendering (CSR), where most of the HTML
+content is composed and rendered in the browser using JavaScript.
+
+## How does HDS support server-side rendering?
+
+For HDS components to work with server-side rendering, in server you need to inject the critical CSS styles
+of the HDS components that are being used into the initially rendered HTML. For hds-core, you must include the styles from
+provided CSS files yourself. For hds-react, HDS provides multiple options, that we will cover next.
+
+<Notification label="Critical CSS" className="siteNotification">
+Critical CSS is the CSS applied to above-the-fold elements. It is the CSS of the immediately visible content in the browser
```

Comment:
A lot of repetition of the word CSS. Maybe something like this is a bit better: "Critical CSS is applied to the above-the-fold elements. It provides the styles for the immediately visible content in the browser."

Reply:
- [ ] Refactor to: Critical CSS is applied to the above-the-fold elements. It provides the styles for the immediately visible content in the browser...